Makefile

```diff
@@ -75,7 +75,7 @@ upload-codedeploy-artifact: check-env-vars
 	aws s3 cp --region eu-west-1 target/notifications-api.zip s3://${DNS_NAME}-codedeploy/notifications-api-${DEPLOY_BUILD_NUMBER}.zip
 
 .PHONY: test
-test: venv ## Run tests
+test: venv generate-version-file ## Run tests
```

Comment:
Ok, it won't hurt if we add it here, but obviously anyone should run build (which generates the version file) and then test. We just don't want to add the build here as a dependency.